doc/releases/luminous.rst

```diff
@@ -1,3 +1,157 @@
+v12.2.11 Luminous
+=================
+
+This is the eleventh bug fix release of the Luminous v12.2.x long term stable
+release series. We recommend that all users upgrade to this release. Please note
+the following precautions while upgrading.
+
+Notable Changes
+---------------
+
+A previous release, v12.2.9, introduced the PG hard-limit patches which were
+found to cause an issue in certain upgrade scenarios, and 12.2.10 release was
+expedited to revert those patches. If you already successfully to v12.2.9, then
```

Comment:
If you already successfully installed v12.2.9, . . .

Reply:
tnx